griptape/tools/prompt_video_generation/tool.py

```diff
@@ -0,0 +1,46 @@
+from __future__ import annotations
+
+from typing import TYPE_CHECKING
+
+from attrs import define, field
+from schema import Literal, Schema
+
+from griptape.tools import BaseVideoGenerationTool
+from griptape.utils.decorators import activity
+
+if TYPE_CHECKING:
+    from griptape.artifacts import ErrorArtifact, VideoArtifact
+    from griptape.drivers import DreamMachineVideoGenerationDriver
+
+
+@define
+class PromptVideoGenerationTool(BaseVideoGenerationTool):
```

Comment:
this doesnt need to be two classes, just inherit from `BaseTool` and make `VideoGenerationTool`